### PR TITLE
User token and controller

### PR DESCRIPTION
**Pull request by Dru01**
API for user login by token and user controller

### CODE REVIEW

File: GasolinerasVIP.API/Controllers/UsersController.cs [hunk 1]
```diff
@@ -14,7 +18,11 @@ public class UsersController : ControllerBase
         private readonly UserManager<IdentityUser> userManager;
```
Comment (JorgeSC31) [inline GasolinerasVIP.API/Controllers/UsersController.cs:19]: I think this should be encapsulated in a class.

All the functions.

Comment (Dru01) [reply GasolinerasVIP.API/Controllers/UsersController.cs:19]: They all ready are, in UsersController class.

**Review by Dru01 — COMMENTED** [review]
The class which si encapsulating averything is UsersController

**Review by JorgeSC31 — APPROVED** [review]
LGTM

**Pull request merged**